main.py

```diff
@@ -32,8 +37,8 @@ class Main(KytosNApp):
     # that is not the case (i.e., overlapping replies), we skip up to X
     # cycles before cleaning up pending requests and getting a fresh start
     _multipart_replies_xids = {}
-    _multipart_replies_flows = {}
-    _multipart_replies_ports = {}
```

Comment:
FYI, these two locks are no longer needed since the points they were protecting was because of Thread preemption, with `async` in the parts that they're being used shouldn't be an issue anymore.